content/getting-started/quick-start-my-first-webhook/step-2.md

```diff
@@ -0,0 +1,139 @@
+# _Step 2_ | Create your own Symfony app to display event subscriptions data 
+
+[//]: <> (Hey Akeneo colleagues, here is a repo with the working code if you need to test https://github.com/akeneo/events-api-app.)
+
+## Prerequisites
+
+First of all, you'll need a new Symfony microservice app. To do so, you can follow [the Symfony guide](https://symfony.com/doc/current/setup.html) and run the `$ symfony new display_webhook` command.
```

Comment:
```suggestion
First of all, you'll need a new Symfony app. To do so, you can follow [the Symfony guide](https://symfony.com/doc/current/setup.html) and run the `$ symfony new display_webhook` command.
```
"microservice" is a specific type of architecture, it feels weird to use this term here for a simple web app